cmd/kubemark/hollow-node.go

```diff
@@ -21,9 +21,7 @@ import (
 	"errors"
 	goflag "flag"
 	"fmt"
-	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/client-go/util/clientutil"
```

Comment:
nit: move this k8s import to second block

Reply:
I agree, We should do the same sorting for all files.